wagtail/admin/templates/wagtailadmin/pages/listing/_table_headers_explore.html

```diff
@@ -17,24 +17,22 @@
 {% endcomment %}
 
 <tr class="table-headers">
-    {% if show_ordering_column %}
```

Comment:
**Question:** how does one change the order of the page tree without this option?